supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,76 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+.Technical best practices
+
+* Include links only when necessary.
+* Use hyperlinks and use them consistently.
+* Default to followable, crawlable links. Do not use “nofollow” links unless absolutely necessary.
+* Do not use JavaScript links. 
+* Do not use Flash links.
+* Do not use links in Frames or iframes.
+
+
+.Link text best practices
+
+* Contextually describe what the user will find at the target location so that they can decide if they want to leave their current location.
+* Use a concise sentence or sentence fragment as the link text.
+* Avoid bare URLs. Bare URLs are unhelpful because they have no information about the target location.
```

Comment:
Hi @bredamc, wdyt about this addition about Bare URLs based on your comment?

Reply:
Thanks @ndeevy -- I _think_ the main problem with bare URLs from an accessibility POV is that screen readers read them out character by character, which is very tedious for the user? Perhaps it would be best to ask @neal-timpe to review this text?

Reply:
The [success criterion for accessibility of links](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html) says that it's mostly about context. But @bredamc, I agree it is also annoying to hear the URL, which sometimes is encoded, like in a shortened URL, and doesn't provide any context about what might be at that link. So it's annoying because it's often unhelpful.

> The intent of this Success Criterion is to help users understand the purpose of each link so they can decide whether they want to follow the link.

I'd change `Avoid bare URLs` to `Do not use bare URLs`. Avoid gives the impression there may be a time they're okay. In a hyperlink, that's not the case. I think this line might be improved by saying:

>Do not use bare URLs. Bare URLs are unhelpful because they do not provide adequate context about the link target.

Reply:
Great -- many thanks, @neal-timpe!

Reply:
Thank you both.